<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QDialog" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户信息</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qss.qrc">
    <normaloff>:/icons/baiyang32.png</normaloff>:/icons/baiyang32.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>631</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,4">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Secret Key：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_SK">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>181</y>
     <width>681</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,4">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Token：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_token">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>59</y>
     <width>631</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,4">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_username">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>631</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,4">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>API Key：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_APIK">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="confirmBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>90</y>
     <width>631</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,4">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>AppID：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_AppID">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="qss.qrc"/>
 </resources>
 <connections/>
</ui>
